ocs_ci/tests/test_import_all.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf8 -*-
+"""
+This unit test tries to just import every module within ocs_ci module, to
+detect serious errors (eg. a syntax error) or dependency problems.
+"""
+
+import importlib
+import pkgutil
+
+import pytest
+
+
+def list_submodules(module_path, module_prefix):
+    """
+    For given module, return list of full module path names for all submodules
+    recursively.
+    """
+    return [name for _, name, _ in
+            pkgutil.walk_packages(path=[module_path], prefix=module_prefix)]
```

Comment:
```suggestion
    return [
        name for _, name, _ in
        pkgutil.walk_packages(path=[module_path], prefix=module_prefix)
    ]
```

Reply:
Looks like this change was reverted, this can be fixed after this PR is merged tho.